src/api/todoApi.js

```diff
@@ -14,3 +14,8 @@ export const addTodoToDB = async (item) => {
 export const deleteTodoFromDB = (id) => {
   axios.delete(`${config.API_URL}/${id}`, id);
 };
+
+export const updateTodoInDB = async (id, value) => {
+  const response = await axios.put(`${config.API_URL}/${id}`, { value });
```

Comment:
Lets rename 'value' to 'post' object, where we will put 'completed' and/or 'title' fields